aries-site/src/examples/templates/forms/SingleFormFieldExample.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Text,
+  TextInput,
+} from 'grommet';
+
+export const SingleFormFieldExample = () => {
+  const [cancel, setCancel] = React.useState('');
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  const confirmDelete = () => {
+    const doesMatch = cancel.delete === 'DELETE';
+    return doesMatch
+      ? undefined
+      : { message: 'Verification Incorrect', status: 'error' };
+  };
+
+  return (
+    <Box
```

Comment:
Feels like this should be a Card with CardBody? We would get a lot of the padding, elevation, and rounding for free

Reply:
good thought Ill update